### ISSUE TITLE
Forge-Version request

### ISSUE THREAD

**Issue by marlester-dev** [WindClan/oldgl#1]
For compatiblity with some mods I need a forge version please

**Comment by WindClan** [comment]
try ForceGL20
it does the same thing but multiple mod loaders

**Comment by marlester-dev** [comment]
> try ForceGL20 it does the same thing but multiple mod loaders

the problem is that it doesn't function on Forge at all, not even the fork of it called forcegl20 remapped. still getting the same error, and no support provided. it works on Fabric, but not on Forge. The author stated that he will not provide any support for issues in the plugin.

**Comment by WindClan** [comment]
So the fabric version works? Does this mod work and ForceGL20 not?
If so I'll try and make a forge version

Tell me the versions of mc (i cant get multi version forge) and I'll try

**Comment by marlester-dev** [comment]
Yes, ForceGL20 doesn't work on Forge. Here are some reports: [1](https://www.reddit.com/r/MinecraftHelp/comments/16dx83t/comment/jzxv1r6/), [2](https://github.com/coredex-source/ForceGL20-1.2x/issues/1).
Since were not able to do Forge with multiple versions, I think it would be good to do Forge 1.18.2, 1.19.2, 1.20.1 and 1.21. If you want a larger version I would consider 1.17.1, 1.18.2, 1.19.2, 1.20.1, 1.20.4, 1.20.6 and 1.21. No pressure of course. Thank you in advance.

**Comment by gdmeunier** [comment]
I want to mention the following Modrinth links:

**Load FabricMC mods** using **Forge** modloader:
- https://modrinth.com/mod/forgified-fabric-api

**Load Forge mods** using **FabricMC** modloader:
- https://modrinth.com/mod/fabricated-forge

Your OldGL mod is basically a game optimization mod, and should be supported by Forgified.

If not, then you might ask their developers if OldGL cans be supported.
They will surely be interested since this project is very rare in the Minecraft world, only very few mods for OpenGL downgrading.

Hope it helps!

_One edit:_

See the following about NeoForge (a Forge fork):
- https://github.com/neoforged/NeoForge/discussions/58#discussion-5446904

```
At present, my action identifies NeoForge mods as Forge mods, which is in line with what was stated on your website: At the present time, mods built by either system should be intercompatible between forks.
```
